rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.75"
+channel = "1.78"
```

Comment:
We need to modify the `flake.nix` to handle this. Is there anything that actually depends on this? I assume this forced by `alloy`?

Reply:
I set to 1.76 because of Alloy. I didn't know about nix overlay.

Reply:
Is it possible to stay on `1.75`? I ask for hyperlane fork comptability. We need `movement` to run in the same env as `hyperlane` binaries, which is on `1.75`.

Reply:
Alloy need 1.76 that why I change the version. To allow using different compilers, we can remove settlement/mcr/client from the workplace to have its own build and own compiler version.